extensions/guacamole-auth-totp/src/main/resources/translations/fr.json

```diff
@@ -0,0 +1,34 @@
+{
+
+    "DATA_SOURCE_TOTP" : {
+        "NAME" : "TOTP TFA Backend"
+    },
+
+    "LOGIN" : {
+        "FIELD_HEADER_GUAC_TOTP" : ""
+    },
+
+    "TOTP" : {
+
+        "ACTION_HIDE_DETAILS" : "Masquer",
+        "ACTION_SHOW_DETAILS" : "Montrer",
+
+        "FIELD_HEADER_ALGORITHM"  : "Algorithme:",
+        "FIELD_HEADER_DIGITS"     : "Chiffres:",
+        "FIELD_HEADER_INTERVAL"   : "Intervalle:",
+        "FIELD_HEADER_SECRET_KEY" : "Clé secrète:",
+
+        "FIELD_PLACEHOLDER_CODE" : "Code d'authentification",
+
+        "INFO_CODE_REQUIRED"       : "Veuillez entrer le code d'authentification pour vérifier votre identité.",
+        "INFO_ENROLL_REQUIRED"     : "L'authentification multi-facteurs a été activée pour votre compte.",
+        "INFO_VERIFICATION_FAILED" : "La vérification a échoué. Veuillez réessayer.",
+
+        "HELP_ENROLL_BARCODE" : "Pour terminer votre processus d'inscription, scannez le code-barre ci-dessous avec l'application deux-facteurs sur votre téléphone ou votre appareil",
+        "HELP_ENROLL_VERIFY"  : "Après avoir scanné le code-barre, saisissez les {DIGITS} chiffres du code d'authentification affichés pour terminer votre inscription.",
```

Comment:
A small thing, but I believe these items should be ordered alphabetically, so the `HELP_` items should above the `INFO_` items.

Reply:
I have used the same order that I found in English file for easier comparison. If order is important, all translations files need to be updated...